ert_shared/storage/endpoints/responses.py

```diff
@@ -101,6 +106,7 @@ async def read_response_by_id(*, db: Session = Db(), ensemble_id: int, id: int):
             obs_values = list(observation.values)
             obs_stds = list(observation.errors)
             obs_data_indexes = list(observation.data_indices)
+            obs_key_indexes = list(observation.key_indices)
```

Comment:
Should this new key in the json also have a corresponding test ?

Reply:
Doesn't this https://github.com/equinor/ert/pull/1375/files#diff-dff280db3a36b0c8a92239a8f48346cc0178f5f00c02a0d0b360057544828e89 suffice or you had some other in mind?

Reply:
naaah, i was thinking more updating this test with the new property 😃 
```
def test_observation(app_client):
    ens = app_client.get(p.ensemble(1)).json()
    expected = {
        "active_mask": [True, False],
        "data_indexes": [2, 3],
        "key_indexes": [0, 3],
        "std": [1, 3],
        "values": [10.1, 10.2],
    }
...
...
```

Reply:
Forget it - i was fooled by the way github collapsed code! 👍  Looks good !